src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -2891,6 +2891,39 @@ def test_aks_create_with_windows_gmsa(self, resource_group, resource_group_locat
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_with_default_network(self, resource_group, resource_group_location):
+        # reset the count so in replay mode the random names will start with 0
+        self.test_resources_count = 0
+        # kwargs for string formatting
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'location': resource_group_location,
+            'resource_type': 'Microsoft.ContainerService/ManagedClusters',
+            'ssh_key_value': self.generate_ssh_keys(),
+        })
+
+        # create
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --location={location} ' \
+                     '--ip-families IPv4 --pod-cidr 172.126.0.0/16 --service-cidr 172.56.0.0/16 ' \
+                     '--dns-service-ip 172.56.0.10 --pod-cidrs 172.126.0.0/16 --service-cidrs 172.56.0.0/16 ' \
+                     '--network-plugin kubenet --ssh-key-value={ssh_key_value}'
```

Comment:
It seems that the new option `--load-balancer-managed-outbound-ipv6-count` is not covered by any live (E2E) test case. Please add one and test both `aks create` and `aks update`.

Reply:
currently, we do not have the toggle on for dual-stack clusters so any e2e test would fail. That should be changing shortly. My plan was to come back and add a test once it is enabled in canary (which should be by Monday)

Reply:
these tests are now added for the ipv6-count